tests/DanBettles/Telex/TelexTest/telephone_numbers.csv

```diff
@@ -5040,7 +5040,8 @@
 1,+33 8 25 08 71 56
 1,+33 2 40 44 44 44
 1,+33 8 10 44 44 50
-0,+33 2 40 + 75 75
+1,+33 2 40 + 75 75
+1,+33 693+633333
```

Comment:
Could you add a couple more tests to see what now happens when you have a case like `+33 693+633333/+33 789 456123`, please?  Thanks.